plugin/storage/cassandra/spanstore/dbmodel/tag_filter_exact_match.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2019 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dbmodel
+
+import (
+	"github.com/jaegertracing/jaeger/model"
+)
+
+// ExactMatchTagFilter filters out all tags in its tags slice
+type ExactMatchTagFilter struct {
+	tags        map[string]struct{}
+	dropMatches bool
+}
+
+// NewExactMatchTagFilter creates a ExactMatchTagFilter with the provided tags.  Passing
+// dropMatches true will exhibit blacklist behavior.  Passing dropMatches false
+// will exhibit whitelist behavior.
+func NewExactMatchTagFilter(tags []string, dropMatches bool) ExactMatchTagFilter {
```

Comment:
nit: I would prefer to keep the factory method names as NewBlacklistTagFilter/NewWhitelistTagFilter, even though they return the same type. In this current form it's hard to read at the call sites. E.g. in 6 months would you remember what this means: `NewExactMatchTagFilter(tags, true)`?